src/main/webapp/i18n/en/course.json

```diff
@@ -93,7 +93,8 @@
                 },
                 "messagingEnabled": {
                     "label": "Messaging Enabled",
-                    "tooltip": "Enables messaging between course users in private or public channels, group chats or direct messages. Channels can only be created by instructors and tutors. Users can self-join public channels and must be invited to private channels. Every user can start a private group chat and add other users. A group chat is limited to 10 members. Every user can start a private one-to-one chat with another user. The chats happens in the messaging space of the course."
+                    "tooltip": "Enables messaging between course users in private or public channels, group chats or direct messages. Channels can only be created by instructors and tutors. Users can self-join public channels and must be invited to private channels. Every user can start a private group chat and add other users. A group chat is limited to 10 members. Every user can start a private one-to-one chat with another user. The chats happens in the messaging space of the course.",
```

Comment:
Is the tooltip actually used somewhere?
(might have missed it)

Reply:
Yes , it is used on the course edit view, right next to the "Messaging Enabled" toggle.